src/latent_gp.jl

```diff
@@ -0,0 +1,32 @@
+"""
+    LatentGP(f<:GP, x<:AbstractVector, Φ)
+
+ - `fx` is a `FiniteGP`.
+ - `Φ` is the log likelihood function which maps input of type `Tx` to `Real`.
+    
+"""
+struct LatentGP{T<:AbstractGPs.FiniteGP}
+    fx::T
+    Φ
+    function LatentGP(fx::AbstractGPs.FiniteGP, Φ) where Tx
+        return new{typeof(fx)}(fx, Φ)
+    end    
+end
```

Comment:
```suggestion
struct LatentGP{Tfx<:AbstractGPs.FiniteGP, TΦ}
    fx::Tfx
    Φ::TΦ
end
```

Reply:
Thanks. Didn't notice it to be the default constructor. :sweat_smile: